mmv1/third_party/terraform/services/logging/resource_logging_bucket_config_test.go

```diff
@@ -642,3 +642,52 @@ resource "google_logging_project_bucket_config" "basic" {
 }
 `, context), urlIndexType, statusIndexType)
 }
+
+func TestAccLoggingBucketConfigProject_restrictedFields(t *testing.T) {
+
+	context := map[string]interface{}{
+		"project_name":    "tf-test-" + acctest.RandString(t, 10),
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
+		"bucket_id":       "tf-test-bucket-" + acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccLoggingBucketConfigProject_restrictedFields(context, "jsonPayload"),
```

Comment:
These new fields are updatable, can we add another step that updates them to make sure the update functionality works

Reply:
Thanks, away from home at the moment but will update when I get back